modules/rosa-install-logging-addon.adoc

```diff
@@ -0,0 +1,76 @@
+:_module-type: PROCEDURE
+// Module included in the following assemblies:
+//
+// rosa_logging/rosa-install-logging.adoc
+
+[id="rosa-install-logging-addon_{context}"]
+= Install logging add-on services on {product-title}
+
+{product-tite} (ROSA) does not come preconfigured with a logging solution, but logging can be enabled by manually installing the `cluster-logging-operator` service.
+
+Log forwarding capability is provided through the Amazon CloudWatch add-on service.
+
+.Prerequisites
+
+Request the following link:https://docs.aws.amazon.com/servicequotas/latest/userguide/request-quota-increase.html[quotas from AWS]:
```

Comment:
I'd like to avoid a singular instance of a level one bullet, so I tweaked the wording it a bit to meet the standards. It creates some link redundancy, but it reads better now.